### PR TITLE
Test multiple versions of ruby

### PR DESCRIPTION
**Pull request by backus**
- Runs RuboCop and RSpec on 2.2, 2.3, 2.4
- Runs mutest on 2.2 only

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -26,13 +64,34 @@ steps:
     agents:
       - queue=elastic
 
-  - name: ':skull: Mutest'
-    command: build/run 2.3.3 'rake metrics:mutest'
-    timeout_in_minutes: 10
+  - name: ':rubocop: Rubocop'
```
Comment (dgollahon) [inline .buildkite/pipeline.yml]: We probably only need to run rubocop on `2.2.7`, don't you think? We might get slight conflicts between the versions and really only need to adhere to one.

Unless maybe for adding string literal comments, i guess?

Comment (backus) [inline .buildkite/pipeline.yml]: Yeah good point

Comment (backus) [inline .buildkite/pipeline.yml]: and no I think removing frozen string literals until it works everywhere is preferable

Comment (dgollahon) [inline .buildkite/pipeline.yml]: > and no I think removing frozen string literals until it works everywhere is preferable

I think only if we plan to freeze every string literal manually. If we're not going to, we should at least get the free performance and immutability benefits for 2.3/2.4.

Comment (dgollahon) [inline .buildkite/pipeline.yml]: (which I'm fine with. i'm just saying we should freeze what we can, one way or another).

**Comment by dgollahon** [comment]
This would be good to do, but i'm only offering barebones support/tiny changes to mutest right now and don't plan to follow up here so I'm going to close this.

**Pull request closed without merging**